tests/integrated/test-squash/runtest

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python3
+
+import netCDF4 as nc
```

Comment:
This is tied to `netCDF4` -- is it possible to just use `Datafile`?

Reply:
It anway requires netcdf, the way it is implemented now. Without it becomes far more complicated, as I need to detect the correct file extension etc.
Without that extra code, Datafile could be used, but I see no advantage ...
I prefer netcdf, as it does not load everything immediately.
If you feel strongly about that change, I can change it though.

Reply:
Sorry, I was not very clear, I meant the `netCDF4` python module, which is not always available everywhere and can be a pain to install on certain systems (e.g. see the problems we had getting it working on Travis!). `Datafile` takes care of this, supporting several NetCDF python modules. We should be using it instead of "bare" NetCDF modules like this.

Also, the lazy-loading of NetCDF files is because they are backed by HDF5 which handles that, so if reading HDF5 files via `Datafile` loads everything immediately, that's our bug.

This test should probably also be marked as requiring NetCDF at any rate.